libs/blocks/fragment/fragment.js

```diff
@@ -63,6 +63,14 @@ export default async function init(a) {
   let relHref = localizeLink(a.href);
   let inline = false;
 
+  if (a.parentElement?.nodeName === 'P') {
+    const children = a.parentElement.childNodes;
+    const div = document.createElement('div');
```

Comment:
`createTag` can also be used here.

Reply:
Correct. It's more neat with `createTag `. Made the change 👍